### PR TITLE
FizzBuzz in js

### PR DESCRIPTION
**Pull request by sukeshni**
@shunjikonishi Please review

### CODE REVIEW

File: app/fizzbuzz.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function fizzbuzz(data) {
+	//BEGIN_CHALLENGE
+	var result = [];
```
Comment (shunjikonishi) [inline app/fizzbuzz.js]: Why empty array?
result is not array.

**Comment by shunjikonishi** [comment]
@sukeshni fix me

**Comment by sukeshni** [comment]
@shunjikonishi Please review

**Comment by shunjikonishi** [comment]
LGTM

**Pull request closed without merging**